koku/masu/processor/tasks.py

```diff
@@ -358,7 +358,7 @@ def update_summary_tables(  # noqa: C901
     """
     worker_stats.REPORT_SUMMARY_ATTEMPTS_COUNTER.labels(provider_type=provider).inc()
     task_name = "masu.processor.tasks.update_summary_tables"
-    cache_args = [schema_name, provider, provider_uuid]
+    cache_args = [schema_name, provider, provider_uuid, start_date, end_date]
```

Comment:
I think this leaves us open to running multiple tasks for the same month. How about taking the start date and replacing the day with the first of the month and just passing that in as our one date arg? That way we restrict per month.